### ISSUE TITLE
Sort out all research assets for the project

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/internship#182]
### Overview
We need to have a place where new research team members can get up to speed quickly on everything has been done and where we are going.

### Action Items
- [x] Review all the issues for research 
- [x] Identify questions and add to Questions issue
- [x] Review the Research Page on Wiki
- [ ] Discuss Formatting options with PM

### Resources/Instructions

- [All Research Issues minus branding ](https://github.com/hackforla/internship/issues?q=is%3Aissue+label%3A%22role%3A+UI%2FUX+research%22+-label%3A%22feature%3A+branding%22+is%3Aopen)
- https://github.com/hackforla/internship/issues/59
- WIKI: https://github.com/hackforla/internship/wiki
  - https://github.com/hackforla/internship/wiki/Research-Process
- Hack for LA website team [WIKI Template for Research](https://github.com/hackforla/website/wiki/research-wiki-template)

**Comment by Rabia2219** [comment]
@christine-el 
Hi...this is the issue you should post your update in...thanks :)

**Comment by christine-el** [comment]
Update for 5/26 All Team Meeting:  Last week Bonnie and I began sorting through the research assets.  

1.  We created a new page on the Wiki organizing the assets here:  https://github.com/hackforla/internship/wiki/Research-Output-Overview.
2.    I created 4 labels for the outstanding issues and categorized the issues using these labels: RP1 (Research Project 1), RP2, RP3, RP4.  RP1=IS21 Interns; RP2=transferable skills; RP3=LinkedIn Recommendation; RP4=hypothesis/STEM education research.

Next Steps:
1.) Bonnie and I need to schedule another 2-hour meeting to finish sorting the research assets and begin outlining the next steps.
2.) We should go over label RP4 to make sure it's correct and create this category on the WIKI page.
